src/plugins/commit-message/index.js

```diff
@@ -65,73 +35,41 @@ const ERR_CODES_MAPS = {
  */
 function checkCommitMessage(message) {
     const commitTitle = message.split(/\r?\n/)[0];
-    const errCode = [];
+    const errors = [];
 
     if (message.startsWith("Revert \"")) {
-        return [true, []];
+        return { errors };
     }
 
-    let isValid = true;
-
     // First, check tag and summary length
     if (!TAG_REGEX.test(commitTitle)) {
-        isValid = false;
-        errCode.push("NON_MATCHED_TAG");
+        errors.push("NON_MATCHED_TAG");
     }
 
     // Check if there is any whitespace after the <Tag>:
     if (!TAG_SPACE_REGEX.test(commitTitle)) {
-        isValid = false;
-        errCode.push("SPACE_AFTER_TAG_COLON");
+        errors.push("SPACE_AFTER_TAG_COLON");
+    }
+
+    if (!UPPERCASE_TAG_REGEX.test(commitTitle)) {
+        errors.push("NON_UPPERCASE_FIRST_LETTER_TAG");
     }
 
     if (!(commitTitle.length <= MESSAGE_LENGTH_LIMIT)) {
-        isValid = false;
-        errCode.push("LONG_MESSAGE");
+        errors.push("LONG_MESSAGE");
     }
 
     // Then, if there appears to be an issue reference, test for correctness
-    if (isValid && POTENTIAL_ISSUE_REF_REGEX.test(commitTitle)) {
+    if (errors.length === 0 && POTENTIAL_ISSUE_REF_REGEX.test(commitTitle)) {
         const issueSuffixMatch = CORRECT_ISSUE_REF_REGEX.exec(commitTitle);
 
         // If no suffix, or issue ref occurs before suffix, message is invalid
         if (!issueSuffixMatch || POTENTIAL_ISSUE_REF_REGEX.test(commitTitle.slice(0, issueSuffixMatch.index))) {
-            isValid = false;
-            errCode.push("WRONG_REF");
+            errors.push("WRONG_REF");
         }
     }
 
-    return [isValid, errCode];
-}
-
-/**
- * Create a comment message body with the error details
- * @param {Array<string>} errCodes - list of the error codes
- * @param {boolean} isTitle - whether it is for title or the commit message
- * @param {string} username - username of the PR author
- * @returns {string} the message to comment
- * @private
- */
-function commentMessage(errCodes = [], isTitle = false, username) {
-    const errorMessages = [];
-
-    errCodes.forEach(err => {
-        if (ERR_CODES_MAPS[err]) {
-            errorMessages.push(ERR_CODES_MAPS[err]);
-        }
-    });
-
-    return `Hi @${username}!, thanks for the Pull Request
-
-The ${isTitle ? "PR title" : "first commit message"} format is not according to our format.
-
-#### Here are the following errors
-
-${errorMessages.join("\n")}
-
-Read more about contributing to Eslint [here](https://eslint.org/docs/developer-guide/contributing/)
-`;
-
+    return { errors };
```

Comment:
If we are only returning errors, maybe this should just return the errors array instead of an object?